src/detectors/FEMC/FEMC.cc

```diff
@@ -22,32 +22,39 @@ extern "C" {
         using namespace eicrecon;
 
         InitJANAPlugin(app);
-
+        // Make sure digi and reco use the same value
+        decltype(CalorimeterHitDigiConfig::capADC)        EcalEndcapP_capADC = 16384; //16384, assuming 14 bits. For approximate HGCROC resolution use 65536
+        decltype(CalorimeterHitDigiConfig::dyRangeADC)    EcalEndcapP_dyRangeADC = 3 * dd4hep::GeV;
+        decltype(CalorimeterHitDigiConfig::pedMeanADC)    EcalEndcapP_pedMeanADC = 200;
+        decltype(CalorimeterHitDigiConfig::pedSigmaADC)   EcalEndcapP_pedSigmaADC = 2.4576;
+        decltype(CalorimeterHitDigiConfig::resolutionTDC) EcalEndcapP_resolutionTDC = 10 * dd4hep::picosecond;
         app->Add(new JChainMultifactoryGeneratorT<CalorimeterHitDigi_factoryT>(
           "EcalEndcapPRawHits", {"EcalEndcapPHits"}, {"EcalEndcapPRawHits"},
           {
-            .eRes = {0.00340 * sqrt(dd4hep::GeV), 0.0009, 0.0 * dd4hep::GeV}, // (0.340% / sqrt(E)) \oplus 0.09%
+            .eRes = {0.11333 * sqrt(dd4hep::GeV), 0.03, 0.0 * dd4hep::GeV}, // (0.340% / sqrt(E)) \oplus 0.09% divided by corrMeanScale = 0.03
             .tRes = 0.0,
-            .capADC = 65536, //2^16  (approximate HGCROC resolution) old 16384
-            .capTime = 100, // given in ns, 4 samples in HGCROC
-            .dyRangeADC = 3 * dd4hep::GeV,
-            .pedMeanADC = 100,
-            .pedSigmaADC = 0.7,
-            .resolutionTDC = 10 * dd4hep::picosecond,
+            .threshold = 0.0,
+             // .threshold = 15 * dd4hep::MeV,
+            .capADC = EcalEndcapP_capADC,
+            .capTime =  100, // given in ns, 4 samples in HGCROC
+            .dyRangeADC = EcalEndcapP_dyRangeADC,
+            .pedMeanADC = EcalEndcapP_pedMeanADC,
+            .pedSigmaADC = EcalEndcapP_pedSigmaADC,
+            .resolutionTDC = EcalEndcapP_resolutionTDC,
             .corrMeanScale = 0.03,
           },
           app   // TODO: Remove me once fixed
         ));
         app->Add(new JChainMultifactoryGeneratorT<CalorimeterHitReco_factoryT>(
           "EcalEndcapPRecHits", {"EcalEndcapPRawHits"}, {"EcalEndcapPRecHits"},
           {
-            .capADC = 65536,
-            .dyRangeADC = 3. * dd4hep::GeV,
-            .pedMeanADC = 100,
-            .pedSigmaADC = 0.7,
-            .resolutionTDC = 10 * dd4hep::picosecond,
-            .thresholdFactor = 5.0,
-            .thresholdValue = 2.0,
+            .capADC = EcalEndcapP_capADC,
+            .dyRangeADC = EcalEndcapP_dyRangeADC,
+            .pedMeanADC = EcalEndcapP_pedMeanADC,
+            .pedSigmaADC = EcalEndcapP_pedSigmaADC,
+            .resolutionTDC = EcalEndcapP_resolutionTDC,
+            .thresholdFactor = 0.0,
+            .thresholdValue = 2731.0,
```

Comment:
Is this number a bit too high? `3. GeV / 16384 * 2731.0 = 0.5 GeV`

```suggestion
            .thresholdValue = 2731.0, // 0.5 GeV
```

Reply:
Not even 0.5 GeV. We would then divide this by the sampling fraction and arrive to $\approx$ 16.7 GeV.